docs/notebooks/image_to_image_arcface.md

```diff
@@ -0,0 +1,224 @@
+---
+jupyter:
+  jupytext:
+    text_representation:
+      extension: .md
+      format_name: markdown
+      format_version: '1.3'
+      jupytext_version: 1.14.1
+  kernelspec:
+    display_name: .venv
+    language: python
+    name: python3
+---
+
+<!-- #region id="p8jc8EyfruKw" -->
+# Image-to-Image Search with ArcFaceLoss
+
+<a href="https://colab.research.google.com/drive/1ZS9FmnR9FzO_JYGCPazFM7TcMNQl51xM?usp=sharing"><img alt="Open In Colab" src="https://colab.research.google.com/assets/colab-badge.svg"></a>
+
+Searching visually similar images with image queries is a very popular use case. However, using pre-trained models does not deliver the best results – the models are trained on general data that lack the particularities of your specific task. Here's where Finetuner comes in! It enables you to accomplish this easily.
+
+This guide will demonstrate how to fine-tune a ResNet model using the `ArcFaceLoss` function.
+
+*Note, please consider switching to GPU/TPU Runtime for faster inference.*
+
+## Install
+<!-- #endregion -->
+
+```python id="VdKH0S0FrwS3"
+!pip install 'finetuner[full]'
+```
+
+<!-- #region id="7EliQdGCsdL0" -->
+## Task
+
+More specifically, we will fine-tune ResNet50 on [Stanford Cars Dataset](http://ai.stanford.edu/~jkrause/cars/car_dataset.html).
+This dataset consists of 196 classes across 16184 docuemnts in total.
```

Comment:
Here it should be explained a bit more what are the documents and classes in this specific datasets, i.e., images of cars and their brands